src/json_deser.c

```diff
@@ -401,3 +414,140 @@ corto_int16 json_deserialize(corto_object o, corto_string s) {
     }
     return -1;
 }
+
+/*
+ * This method changes the id!
+ */
+static void json_splitParentAndName(char* id, char** parentName, char** name)
+{
+    char *ptr = strrchr(id, '/');
+    if (ptr) {
+        *name = ptr + 1;
+        *parentName = id;
+        *ptr = '\0';
+    } else {
+        *name = id;
+        *parentName = NULL;
```

Comment:
Change this to `parentName = "/";` so you can get rid of the if-condition in `json_declareObjectByParentNameNameTypeName`